### DISCUSSION TITLE
How to get a controller attribute from middleware

### DISCUSSION THREAD

**Discussion by MartkCz** [spiral/spiral#1112] [Q&A]
Hello,

the question is simple, but I haven't found any way to do it. I have the following controller code:

```php
#[Cache(['...options'])]
public function action() { ... }
```

How to get the Cache attribute in the middleware?

**Comment by butschster** [comment]
If you want to handle PHP attributes, use interceptors.
Check out the details here: https://spiral.dev/docs/http-interceptors
